egs/wsj/s5/utils/validate_data_dir.sh

```diff
@@ -44,6 +45,12 @@ if [ ! -d $data ]; then
   exit 1;
 fi
 
+if [ -f $data/images.scp ]; then
+  cmd=${cmd/--no-wav/--no-image}  # change --no-wav to --no-image, if supplied
```

Comment:
I think you should just remove the --no-wav rather than adding  --no-image.  If someone added --no-image, that implies they knew they would be calling the image version, in which case they could just call it directly.